sdk/tests/client/input_selection/account_outputs.rs

```diff
@@ -1269,6 +1274,10 @@ fn account_burn_should_validate_account_sender() {
     .select()
     .unwrap();
 
+    assert_eq!(
+        selected.transaction.capabilities(),
+        &TransactionCapabilities::from([TransactionCapabilityFlag::DestroyAccountOutputs])
+    );
```

Comment:
There is another Burn in `account_burn_should_validate_account_address`, do you want to add a check for it?